gateway/conf/nginx.conf.liquid

```diff
@@ -69,119 +68,19 @@ http {
   {%- endcapture -%}
 
   {% for file in "http.d/*.conf" | filesystem %}
+    ## include {{ file }}
```

Comment:
Why is this needed?

Reply:
It is not really needed, but it is useful to see the file that was included in the rendered template.

Reply:
:+1: